bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -35,9 +35,9 @@ spec:
     official support.
 
     ### Pre-requisites
-    * A Red Hat OpenShift Container Platform for Windows Containers subscription [subscription page](https://access.redhat.com/support/policy/updates/openshift#windows)
-    * OCP 4.10 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
-    * [WMCO prerequisites](https://docs.openshift.com/container-platform/4.10/windows_containers/understanding-windows-container-workloads.html#wmco-prerequisites__understanding-windows-container-workloads)
+    * A Red Hat OpenShift Container Platform for Windows Containers [subscription](https://access.redhat.com/support/policy/updates/openshift#windows)
+    * OCP 4.11 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
+    * [WMCO prerequisites](https://docs.openshift.com/container-platform/latest/windows_containers/understanding-windows-container-workloads.html#wmco-prerequisites__understanding-windows-container-workloads)
```

Comment:
What was the thought around switching this from a specific version to `latest`?  In the future, is there a desire for a user installing 4.11 to be linked to the 4.12 prerequisites, for example?

Reply:
I was thinking of reducing the number of places to change the version going forward, but you bring up a good point. Pointing to newer docs may be confusing. I'll keep this versioned.

Reply:
There were a couple other references to `latest` pages in the CSV. I changed those to be versioned too.